community-operators/mongodb-enterprise-upstream/mongodb.package.yaml

```diff
@@ -0,0 +1,6 @@
+packageName: mongodb-enterprise
```

Comment:
`packageName` should match the name of the dir that these items are being placed in. In this case, it should be `mongodb-enterprise-upstream`. If moved to the upstream folder as @SamiSousa suggested, the dir could be changed to `mongodb-enterprise`.

Reply:
Actually the packageName and directory should be `mongodb-enterprise-community`
The kubernetes version can be as @awgreene described.